de.dlr.sc.virsat.model.extension.tests.test/src/de/dlr/sc/virsat/model/extension/tests/model/TestCategoryIntrinsicArrayStaticTest.java

```diff
@@ -1,170 +1,186 @@
-		addBeans.add(property1);
-		addBeans.add(property2);
-		addBeans.add(property3);
-		
-		arrayStatic.addAll(addBeans);
-	}
-
-	@Test(expected = UnsupportedOperationException.class)
-	public void testAddBeanType() {
-		BeanPropertyString property1 = createNewStringProperty();
-		
-		assertEquals("List has one items", LIST_WITH_STATIC_SIZE, arrayStatic.size());
-			
-		arrayStatic.add(property1);		
-	}
-
-	@Test
-	public void testAddBeanTypeCommand() {
-		BeanPropertyString property1 = createNewStringProperty();
-		
-		assertEquals("List has one items", LIST_WITH_STATIC_SIZE, arrayStatic.size());
-			
-		Command command = arrayStatic.add(editingDomain, property1);
-		assertEquals("Command cannot be executed", UnexecutableCommand.INSTANCE, command);
-	}
-	
-	@Test(expected = UnsupportedOperationException.class)
-	public void testAddIntBeanType() {
-		BeanPropertyString property1 = createNewStringProperty();
-
-		assertEquals("List has one items", LIST_WITH_STATIC_SIZE, arrayStatic.size());
-		
-		arrayStatic.add(1, property1);
-	}
-	
-	@Test(expected = UnsupportedOperationException.class)
-	public void testClear() {
-		arrayStatic.clear();
-	}
-
-	@Test(expected = UnsupportedOperationException.class)
-	public void testRemoveAll() {
-		BeanPropertyString property1 = createNewStringProperty();
-		BeanPropertyString property3 = createNewStringProperty();
-
-		assertEquals("List has one items", LIST_WITH_STATIC_SIZE, arrayStatic.size());
-		
-		List<BeanPropertyString> removeBeans = new ArrayList<>();
-		removeBeans.add(property1);
-		removeBeans.add(property3);
-		
-		arrayStatic.removeAll(removeBeans);
-	}
-
-	@Test(expected = UnsupportedOperationException.class)
-	public void testRemoveInt() {
-		arrayStatic.remove(1);
-	}
-
-	@Test(expected = UnsupportedOperationException.class)
-	public void testRemoveObject() {
-		BeanPropertyString propertyOne = createNewStringProperty();
-
-		arrayStatic.remove(propertyOne);
-	}
-	
-	@Test
-	public void testRemoveObjectCommand() {
-		BeanPropertyString property1 = createNewStringProperty();
-		
-		assertEquals("List has one items", LIST_WITH_STATIC_SIZE, arrayStatic.size());
-			
-		Command command = arrayStatic.remove(editingDomain, property1);
-		assertEquals("Command cannot be executed", UnexecutableCommand.INSTANCE, command);
-	}
-
-	@Test(expected = UnsupportedOperationException.class)
-	public void testRetainAll() {
-		BeanPropertyString property1 = createNewStringProperty();
-		BeanPropertyString property3 = createNewStringProperty();
-		
-		List<BeanPropertyString> retainBeans = new ArrayList<>();
-		retainBeans.add(property1);
-		retainBeans.add(property3);
-		
-		arrayStatic.retainAll(retainBeans);
-	}
-
-	@Test(expected = UnsupportedOperationException.class)
-	public void testSetIntBeanType() {
-		BeanPropertyString property3 = createNewStringProperty();
-
-		arrayStatic.set(1, property3);
-	}
-}
+/*******************************************************************************
```

Comment:
The whole file is marked as changed - probably wrong line endings? Let's revert it and only have meaningful changes to have a cleaner history

Reply:
I think I deleted some trailing tabs -.- Now you can see the changes.